### ISSUE TITLE
Material You color changes also reset other custom colors set by the user

### ISSUE THREAD

**Issue by AlertSubject** [Docile-Alligator/Infinity-For-Reddit#1268]
### Checklist

- [X] I have used the search function for [open](https://github.com/Docile-Alligator/Infinity-For-Reddit/issues) **and** [closed](https://github.com/Docile-Alligator/Infinity-For-Reddit/issues?q=is%3Aissue+is%3Aclosed) issues to see if someone else has already submitted the same bug report.
- [X] I will describe the problem with as much detail as possible.
- [X] If the bug only occurs with a certain link, post, image..., I will include the URL.

### App version

5.4.0.

### Where did you get the app from

F-Droid

### Android version

OxygenOS

### Device model

OnePlus9pro

### First occurred

About 4 weeks ago

### Steps to reproduce

1. Customize some colors inside the material You deisgn (or examaple the color of collapsed comments)
2. Change the Wallpaper and wait for the Material You color to change

### Expected behaviour

The accent color should change according to the wallpaper color but other customizations should remain the same (collapsed comments color)

### Current behaviour

The accent color changes but other colors get reset to default as well, even if they have been modified before

Currently I have to back up all local settings, change the wallpaper, import the old backup and then I can use the app as before

**Comment by Wladefant** [comment]
Isn`t that how material you works? @Docile-Alligator knows more about the issue.

**Comment by AlertSubject** [comment]
> Isn`t that how material you works? @Docile-Alligator knows more about the issue.

Yes I should change the accent colors of the header bar and background tint, as far as I know
But it always resets the other things I've customized after enabling material you

For example the color of collapsed comments
It's annoying to have that color being reset all the time

**Comment by Docile-Alligator** [comment]
I need to think about this.
